docs/authorization-configuration.md

```diff
@@ -136,26 +136,21 @@ auth: {
 - **default**: `60` seconds
 #### thirdPartyCookiesScriptLocation
 - **type**: string
-- **description**: is the URL to the page containing third-party cookies support check. For details, see [Third-party cookies and silent token refresh section](#Third-party-cookies-and-silent-token-refresh).
+- **description**: is the URL to the page containing third-party cookies support check. For details, see [Silent token refresh section](#silent-token-refresh).
 #### userInfoFn
 - **type**: function
 - **description**: provides a function to get user information. It returns a promise of a **userinfo** object which can contain **name**, **email** and **picture** (value is a URL to the image). **Name** or **email** are displayed in the profile drop-down menu and the user’s profile picture is displayed in the top navigation.
 #### profileStorageInterceptorFn
 - **type**: function
 - **description**: provides a function to mutate the profile values of the [JSON Web Token (JWT)](https://jwt.io) before it gets stored in Luigi. It allows the removal of values like **email** to comply with data privacy restrictions. Since it is async, you could use it to enrich the profile data, but it should not get mixed up with **userInfoFn** which is specifically designed to define user information.
 
-### Third-party cookies and silent token refresh
+### Silent token refresh
 
 The OpenID Connect configuration allows you to specify the **automaticSilentRenew** option. When set to `true`, Luigi attempts to automatically renew the token in the background before it expires. Be aware that this mechanism requires the browser to support [third-party cookies](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#Third-party_cookies).
 
-To detect whether the user's browser supports the mechanism, use the script in the [`third-party-cookies`](https://github.com/SAP/luigi/tree/main/core/third-party-cookies) catalog. Deploy this file on a domain different from your main application's and set **thirdPartyCookiesScriptLocation** to the `init.html` file. During initialization, Luigi detects the cookies support and produces a warning in the console if cookies are disabled in the user's browser.
+When Luigi fails to renew the token and then logs the user out, it adds the `?reason=tokenExpired` query parameter to the logout page redirect URL. Use this parameter to set a logout page.
```

Comment:
```suggestion
When Luigi fails to renew the token it logs the user out and  adds the `?reason=tokenExpired` query parameter to the logout page redirect URL. Use this parameter to set a logout page.
```
how should this parameter be used to set the logout page? Luigi adds the token expired parameter and the second one does not exist anymore.

Reply:
yeah, I read the previous version to see if I could find out how it works but it wasn't clear. It may mean something like set's the _parameter_ for the logout page and not the page itself.

Reply:
@ndricimrr I've made a suggestion that I was talking about in the daily. Please check and confirm.

Reply:
@smahati I applied your suggestion - please verify